webapp/views.py

```diff
@@ -715,7 +715,9 @@ def marketo_submit():
 
     if "email" in form_fields:
         # Enrichment data for global enrichment form (id:4198)
-        enrichment_fields = {"email": form_fields["email"]}
+        enrichment_fields = {
+            "email": form_fields["email"],
+            "acquisition_url": referrer}
```

Comment:
Tiny bit of linting:
```suggestion
        enrichment_fields = {
            "email": form_fields["email"],
            "acquisition_url": referrer,
        }
```